Makefile

```diff
@@ -168,6 +173,10 @@ ifeq ($(OSTYPE),osx)
   ALL_CXXFLAGS += -stdlib=libc++ -mmacosx-version-min=10.8
 endif
 
+ifeq ($(UNAME_S),DragonFly)
+  LINKER_FLAGS += -L /usr/local/lib
```

Comment:
/usr/local/lib is explicitly required for DragonFly.

Is there any reason to not make this a default path for every OS?